### PR TITLE
update: Update AKS VHD version

### PR DESCRIPTION
**Pull request by bowang-666**
Update AKS VHD version to 2020.05.06

### CODE REVIEW

File: pkg/api/azenvtypes.go [hunk 1]
```diff
@@ -138,7 +138,7 @@ var (
 		ImageOffer:     "aks",
 		ImageSku:       "aks-ubuntu-1804-gen2-202004",
```
Comment (xuto2) [inline pkg/api/azenvtypes.go:139]: 202005

File: pkg/api/azenvtypes.go [hunk 2]
```diff
@@ -162,15 +162,15 @@ var (
 		ImageOffer:     "aks",
 		ImageSku:       "aks-ubuntu-1604-202004",
```
Comment (xuto2) [inline pkg/api/azenvtypes.go:163]: 202005

File: pkg/api/azenvtypes.go [hunk 3]
```diff
@@ -162,15 +162,15 @@ var (
...
 		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "2020.04.16",
+		ImageVersion:   "2020.05.06",
 	}
 
 	// AKSUbuntu1804OSImageConfig is the AKS image based on Ubuntu 18.04-LTS.
 	AKSUbuntu1804OSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "aks",
 		ImageSku:       "aks-ubuntu-1804-202004",
```
Comment (xuto2) [inline pkg/api/azenvtypes.go:171]: 202005

File: pkg/api/azenvtypes.go (hunk 1 above)
Comment (bowang-666) [reply pkg/api/azenvtypes.go]: updated

**Pull request merged**